### PR TITLE
enh(installation): make user read "Secure your platform" page

### PR DESCRIPTION
**Pull request by cg-tw**
## Description

Moved "Secure your platform" in the TOC to make it part of the chapter on installation. Moved a paragraph from "Web And Post Installation" to "Secure your platform". Added a paragraph in "Installation and first steps".

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [x] 21.10.x (master)

### CODE REVIEW

File: fr/sidebars.json [hunk 1]
```diff
@@ -56,11 +56,11 @@
                 "label": "Installer un Remote Sever",
                 "type": "subcategory"
             },
-            "ceip/what-is-centreon-ceip"
-        ],
-        "Sécurisez votre plate-forme": [
-            "administration/secure-platform",
-            "graph-views/secure-your-map-platform"
+            "Sécurisez votre plate-forme": [
```
Comment (lpinsivy) [inline fr/sidebars.json:59]: I didn't understand this change

File: en/sidebars.json [hunk 2]
```diff
@@ -56,11 +56,11 @@
                 "label": "Installation of a Remote server",
                 "type": "subcategory"
             },
-            "ceip/what-is-centreon-ceip"
-        ],
-        "Secure your platform": [
-            "administration/secure-platform",
-            "graph-views/secure-your-map-platform"
+            "Secure your platform": [
```
Comment (lpinsivy) [inline en/sidebars.json:59]: I didn't understand this change

File: en/administration/secure-platform.md [hunk 3]
```diff
@@ -855,3 +855,8 @@ Centreon event logs are available in the following directories:
 
 Centreon offers to save the configuration of the platform. To do this, go to the
 [`Administration  >  Parameters  >  Backup`](./backup.html) menu.
+
```
Comment (lpinsivy) [inline en/administration/secure-platform.md:858]: For me, the shortcut to the getting started chapter should not be present in the security chapter

File: fr/administration/secure-platform.md [hunk 4]
```diff
@@ -721,3 +721,8 @@ Les journaux des événements Centreon sont disponibles dans les répertoires su
 
 Centreon propose de sauvegarder la configuration de la plateforme. Pour ce faire, accédez au menu
 [`Administration  >  Parameters  >  Backup`](./backup.html).
+
```
Comment (lpinsivy) [inline fr/administration/secure-platform.md:724]: For me, the shortcut to the getting started chapter should not be present in the security chapter

**Review by lpinsivy — CHANGES_REQUESTED** [review]

**Comment by lpinsivy** [comment]
In the past we choose to move the "Administration > Security" chapter to a level 1 to be more visible.
This PR choose to move "Security" to "Installation > Security". So the visibility will be impacted.

**Review by thiuyendang — APPROVED** [review]

**Review by lpinsivy — APPROVED** [review]

**Pull request merged**